mailpoet/assets/js/src/automation/editor/components/header/inserter-toggle.tsx

```diff
@@ -1,9 +1,17 @@
-import { Button, ToolbarItem } from '@wordpress/components';
+import { Button, ToolbarItem as WpToolbarItem } from '@wordpress/components';
 import { useSelect, useDispatch } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 import { plus } from '@wordpress/icons';
 import { storeName } from '../../store';
 
+// Build type for ToolbarItem contains only "as" and "children" properties but it takes all props from
+// component passed to "as" property (in this case Button). So as fix for TS errors we need to pass all props from Button to ToolbarItem.
+// We should be able to remove this fix when ToolbarItem will be fixed in Gutenberg.
+const ToolbarItem = WpToolbarItem as React.ForwardRefExoticComponent<
```

Comment:
@JanJakes I had quite a hard time with some of those Typescript errors. I'm not sure I remember exactly, but in this case, I think yes I used it because it was in the original build types. From that, I saw that I could add another component, and it would accept props. In this place, I did it only for Button.

`/node_modules/@wordpress/components/build-types/toolbar/toolbar-item/index.d.ts`

```ts
/// <reference types="react" />
declare const _default: import("react").ForwardRefExoticComponent<Omit<import("react").HTMLAttributes<any>, "children"> & {
    as?: import("react").ElementType | undefined;
    children?: import("react").ReactNode | ((props: import("react").HTMLAttributes<any> & import("react").RefAttributes<any>) => import("react").ReactElement<any, string | import("react").JSXElementConstructor<any>> | null);
} & import("react").RefAttributes<any>>;
export default _default;
//# sourceMappingURL=index.d.ts.map
```